File: LuxTravel/Model/Models/RateModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Model.Models
{
    public class RateModel
    {
        public Guid Id { get; set; }
        public Guid RoomId { get; set; }
        public Decimal Rate { get; set; }
        public DateTime? FromDate { get; set; }
        public DateTime? ToDate { get; set; }
        public string Name { get; set; }

        public RateTypeModel RateTypeModel { get; set; }
    }
}

File: LuxTravel/Mvc/Areas/Admin/Scripts/Rate/RateModel.js
var RateModel = function(data, parent) {
    var self = this;
    self.Name = ko.observable(data && data.Name ? data.Name : '');
    self.Rate = ko.observable(data && data.Rate ? data.Rate : '');
    self.Icon = ko.observable(getIcon(data.RateTypeModel));
    function getIcon(rateTypeModel) {
        if (rateTypeModel) {
            return rateTypeModel.Icon;
        }
        return'';
    }

};